vulnerabilities/api.py

```diff
@@ -177,9 +183,9 @@ def filter_purl(self, queryset, name, value):
 class PackageViewSet(viewsets.ReadOnlyModelViewSet):
     queryset = Package.objects.all()
     serializer_class = PackageSerializer
-    paginate_by = 50
     filter_backends = (filters.DjangoFilterBackend,)
     filterset_class = PackageFilterSet
+    pagination_class = SmallResultSetPagination
```

Comment:
Why declaring this on the ViewSet instead of globally using the `DEFAULT_PAGINATION_CLASS` setting?